cmd/load-runner/helm.go

```diff
@@ -66,12 +88,84 @@ func (h *helmClient) Uninstall(config *parser.LoadTestConfig) error {
 	return h.executor.run(context.Background(), "helm", args...)
 }
 
-func processHelmEnvVars(args []string, envVars map[string]string) []string {
+func processHelmEnvVars(args []string, envVars map[string]string, logger logger.Logger) []string {
+	args = calculateLoadParameters(args, envVars, logger)
 	for key, value := range envVars {
 		if strings.Contains(value, ",") {
 			value = strings.ReplaceAll(value, ",", "\\,")
 		}
 		args = append(args, "--set", fmt.Sprintf("deployment.env.%s=%s", key, value))
 	}
+
+	return args
+}
+
+func calculateLoadParameters(args []string, envVars map[string]string, logger logger.Logger) []string {
+	resourceCalculation := envVars["RESOURCE_CALCULATION"]
+
+	// If no resource calculation is specified, return args unchanged
+	if resourceCalculation == "" {
+		return args
+	}
+
+	// Validate resource calculation value
+	if resourceCalculation != "auto" && !strings.HasPrefix(resourceCalculation, "overprovision,") {
+		logger.Warn("Invalid RESOURCE_CALCULATION value",
```

Comment:
Please use the non-sugared version `Warnn`

Reply:
As per your comment about errors. Such logs have become errors and thus removed. Will keep the non sugared suggestion for future reference.